src/view_control.cpp

```diff
@@ -1929,6 +1959,41 @@ void View::DrawTurn(DeviceContext *dc, Turn *turn, Measure *measure, System *sys
     dc->EndGraphic(turn, this);
 }
 
+int View::GetOrnamentaccidCode(data_ACCIDENTAL_EXPLICIT ornamentaccid)
```

Comment:
It think the best would be to make is a static method of Accid and then call Accid:: GetOrnamentaccidCode.  I would rename it GetOrnamentAccidGlyph to be inline with other similar methods